cytopipe/cli/cli_docs.py

```diff
@@ -15,13 +15,20 @@
     -c, --max_cores     Maximum number of cores to use for the workflow
                         default=1
     --lock              Enable auto unlocking mechanism for snakemake workflows.
-                        Directory becomes locked when workflow is executed. if 
+                        Directory becomes locked when workflow is executed. if
                         any interruptions has occurred, if True, the directory
                         will be automatically unlocked, else, it will remain
                         locked. Default is False.
     --force             Force re-run of the workflow. This means generated files
                         will be over-written with the outputs produced from the
                         forced re-run
+Available Workflows:
+
+cp_process              Workflow for analyzing cell morphology reads
+                        obtained from CellProfiler
+dp_process              Workflow that analyzes morphological read obtained from
```

Comment:
Can you make these descriptions consistent? The verb is different and it makes me think that the two workflows are doing very different things (when in reality, IIUC, they are doing very similar things just with two different input data types)

Reply:
also, `DeepProfiler` is one word :)